app/assets/stylesheets/views/_landing_page/helpers/_border-style.scss

```diff
@@ -0,0 +1,23 @@
+.mission-1 {
+  border-top: 20px solid #e60067;
```

Comment:
I think you're the first to add these colours so it might help if they were declared as variables for wider use e.g. `$mission-1-colour: #e60067;`

Reply:
Same as above, I'm separating these into a new PR to make the helper available for other components requiring mission specific styles https://github.com/alphagov/frontend/pull/4416.